test/files/neg/t5887.scala

```diff
@@ -0,0 +1,11 @@
+
+trait TheOldCollegeTry {
+
+  // was: value isDefinedAt is not a member of Int
+  // now: required: PartialFunction[Throwable,?]
+  //def f = try ??? catch 22
```

Comment:
I think that's a good error message. Why not test it?

Reply:
That will come back with the change to `catch`, which I'll attempt thricely.

Reply:
Pardon, this is the third time, not so much charmed as cut off at the knees. The next try for catch will be a fourth.